README.md

```diff
@@ -1,161 +1,163 @@
-the factory will load the profile "default". You must specify your customer_id but may 
-
-```yaml
-default:
-  customer_id : "1999999999"
-  auth:
-    # For API Key, you only need the API key
-    api_key : "abcdabcdabcdabcdabcdabcdababcdabcd"
-admin:
-  customer_id : "1999999999" # Customer Id as a string
-  auth:
-    # For OAuth2, you need app_client_id, app_client_secret, auth_url
-    app_client_id : "abcdabcdabcdabcdabcdabcdab"
-    app_client_secret : "abcdabcdabcdabcdabcdabcdababcdabcdabcdabcdabcdabcdab"
-    # This is optional, you can leave this blank in most circumstances
-    auth_url : "https://vectara-prod-YOUR_CUSTOMER_ID.auth.us-west-2.amazoncognito.com/oauth2/token"
-```
-
-#### Multiple Profiles
-You can load other configuration profiles using the property profile on the build command.
-
-```python
-from vectara.factory import Factory
-client = Factory(profile="admin").build()
-```
-
-### Managers vs Direct API methods
-This API was generated by FERN, but in some cases you want to perform operations at a "higher level of abstraction"
-that may comprise multiple method calls. There's also many convenience operations we can perform which are useful that may eventually make it into
-the Vectara API. For example, we can URL encode the doc_id to support multiple languages which isn't (yet) in the
-FERN generated code.
-
-<img src="./resources/managers_seq.png" />
-
-To that end, we have built and tested the following managers on top of the direct API methods. They are located
-in the module vectara.managers. We have the following managers defined which
-can be found on the client.
-
-* [client.corpus_manager](./src/vectara/managers/corpus.py) - Provides abstractions over Corpus Methods
-* [client.upload_manager](./src/vectara/managers/upload.py) - Provides abstractions over file Uploads
-
-## Usage
+## Installation
 
-Using the [implicit configuration](#YAML-Configuration-Format) method we
-can create and run a query as follows.
+Install the library via pip:
 
-```python
-from vectara.factory import Factory
-
-client = Factory().build()
-client.query(
-    query="Am I allowed to bring pets to work?"
-)
+```bash
+pip install vectara
 ```
-More complete examples can be found in the [Getting Started notebooks](./examples/01_getting_started).
-
+## Getting Started
 
-## API Generated Documentation
+### API Generated Documentation
 API reference documentation is available [here](https://vectara.docs.buildwithfern.com/).
 
-## Exception Handling
+### Examples
+Complete examples can be found in the [Getting Started notebooks](./examples/01_getting_started).
+
+### Usage
+
+* Querying the corpora <br />
+  You can use either `api_key` or `client_id` and `client_secret` for authentication.
+    ```python
+    from vectara import SearchCorporaParameters, Vectara
+    client = Vectara(
+        api_key="YOUR_API_KEY", # Use this for API key-based authentication
```

Comment:
This is better but not ideal still. My recommendation is:
1. Have a section about "create a client" that just shows the client=Vectara() piece, and shows it in two different ways (not a single call with "OR" comment - that can be confusing).
2. Then show query (i.e. setup search params, generation params, and the query). But please add some parameters there as an example. IT needs to be a good example where someone can copy/paste and get good results. You can then suggest they can look at docs for more details and options.
3. Chat example.